tests/check-legal-aid/nightwatch.conf.js

```diff
@@ -0,0 +1,41 @@
+var puppeteer = require("puppeteer");
+
+module.exports = {
+  src_folders: "test/specs",
+  output_folder: "reports",
+  custom_assertions_path: "test/custom_assertions",
+  custom_commands_path: "test/commands",
+  globals_path: "test/globals/local.js",
+
+  selenium: {
+    start_process: false
+  },
+
+  test_settings: {
+    default: {
+      launch_url: "http://localhost:5000",
+      selenium_port: 9515,
+      selenium_host: "localhost",
```

Comment:
Do we need `selenium` configuration if we have `selenium.start_process: false` above?

Reply:
We do according to the docs. See http://nightwatchjs.org/gettingstarted#chromedriver